test/unit/test_hpe3par_host.py

```diff
@@ -20,10 +20,12 @@
 from Modules import hpe3par_host as host
 from ansible.module_utils.basic import AnsibleModule as ansible
 import unittest
-
+from hpe3par_sdk.models import Host
 
 class TestHpe3parHost(unittest.TestCase):
 
+    maxDiff = None
```

Comment:
Is this being used anywhere?

Reply:
If there is any assert error, diff between expected output & actual output is truncated after some characters.
If maxDiff is set to None, truncation is not done.